web/src/HUD.scss

```diff
@@ -1,7 +1,5 @@
-@import 'constants';
+@import "constants";
 
 .HUD {
-  display: flex;
-  background-color: $color-navy;
-  min-height: 100vh;
-}
\ No newline at end of file
+  height: 100vh;
```

Comment:
i'm pretty sure the change from min-height to height here is wrong; why would we want to make the HUD exactly 100vh?